src/app/member/components/MemberDetials.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from "@testing-library/user-event";
```

Comment:
can we use `fireEvent` of @testing-library/react because due to this we don't need other library

Reply:
fixed